src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -167,7 +167,9 @@ public int getTotalPersons() {
 
         @Override
         public String getNextUniqueId() {
-            throw new AssertionError("This method should not be called.");
+            // Function needs to be called in order to generate new person model
+            /*throw new AssertionError("This method should not be called.");*/
+            return "";
```

Comment:
Would returning a integer be better here ? this is to mirror the original model behavior